### ISSUE TITLE
ESPnet model zoo

### ISSUE THREAD

**Issue by sw005320** [espnet/espnet#474]
Many people are asking to share the trained models. I think we should make this to some extent.
We could refer some of the following pages, and may create a wiki to share the models.
https://github.com/BVLC/caffe/wiki/Model-Zoo
https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/detection_model_zoo.md

**Comment by sw005320** [comment]
I just prepared a blank wiki page.
https://github.com/espnet/espnet/wiki/Model-Zoo

**Comment by ShigekiKarita** [comment]
I think it is good to provide the output label file `dict=data/lang_1char/${train_set}_units.txt` because it shows what kind of tokens can be predicted by the pretrained model

**Comment by sw005320** [comment]
@ShigekiKarita I agree. 

@sas91, does your model include it? If not, please include it. Also, thanks for your comments. I agree with you that we should prepare some storages about this. About the usage, I think it would be good enough if you prepare it for both resume training and recognition, but I think you could do it with special options in `run.sh` for resume training or some command lines for recognition.

>I have added the CHiME4 model. The model size is around 300 MB so I was not able to add it to github as they have a size restriction of 100 MB, so I added it to google drive. We should figure out a place to upload them. It will also be nice to add the language model (as of now I added only the E2E model and its json file). I am not sure what to add for usage, do you mean the "asr_train" command with all the parameter options or the resume command? I haven't added a snapshot as loading that will also get the optimizer state, I guess it will be better to add a functionality in ESPnet using "model.namedparams()" to initialize parameters as I guess people might be interested in retraining the parameters with a different data, I can give a pull request for this functionality for chainer if you approve.
